vars/artifactsPipeline.groovy

```diff
@@ -77,6 +78,8 @@ def call(Map pipelineParams) {
                             def instance_type = it
                             tasks["${instance_type}"] = {
                                 node(builder.label) {
+                                    // Timeout for the test itself
+                                    timeout(pipelineParams.timeout) {
```

Comment:
please indent the block inside the timeout

Reply:
I have fixed the indent.
The patch w/o the indent is easy to review ;-)